mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomeConfigPrefsBackend.java

```diff
@@ -76,7 +77,11 @@ private State loadDefaultConfig() {
         /* Cliqz start */
         // add Offrz panel, change panels order to be TopSites, history, MyOffrz, bookmarks
         panelConfigs.add(createBuiltinPanelConfig(mContext, PanelType.COMBINED_HISTORY));
-        panelConfigs.add(createBuiltinPanelConfig(mContext, PanelType.MY_OFFRZ));
+        if (BuildConfig.FLAVOR_skin.equals("bond")) {
```

Comment:
Why you are using BuildConfig from Gecko? instead we can use the android one.
BuildConfig.FLAVOR.equal("bond")